### PR TITLE
To DO App : Mile stone : Add, Remove and Edit.

### PR DESCRIPTION
**Pull request by ahmed025566**
In this mile stone you can see that I have made the following : 

- I have created a function to add an objects (task) to the array of tasks.
- I have created a function to display the data from the array.
- I have created a function to update the array tasks whenever the user change any task text.
- I Have created a function to delete any task.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
 <body>
-    
+    <div class="to-do">
+        <div class="today">
+            <p class="text-intro">Today's To Do</p>
+            <i class="fa-sharp fa-solid fa-rotate"></i>
+        </div>
+        <form action="submit" class="form">
+            <input type="text" class="task-text" placeholder="Add to your list ...">
+            <button type="button" class="add-button">Add</button>
```
Comment (hiromataba) [inline dist/index.html:20]: - [ ] Please Fix your button , it is not submitting anything for now on click I think by changing its type to `submit` would fix the issue 


https://user-images.githubusercontent.com/75126481/236030286-91e8fcb3-3d72-454a-aaa1-d64cb24e76f0.mov

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @ahmed025566    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -123,7 +125,6 @@ Using Github Pages
 
 > The following are the future features that will be added to the project.
 
-- [ ] **[Iw-will-has-a-function-to-drag-and-drop-the-task]**
```
Comment (KanzaTahreem) [inline README.md:126]: - [ ] I noticed that the Future Feature section is empty, however, one feature was present before. Please make sure to revert this change and add 1-3 future features in the respective section ✨🛠

README before |  README now
---| ---| 
image | image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @ahmed025566 🙋‍♀️

Good job so far implementing the changes suggested by the previous reviewer, I really appreciate the effort you put in! **_However, there are few things that still need to be fixed. But you are almost there!_** 💪 🏁

so_close

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good branch name
- ✅ Great job with CSS
- ✅ Descriptive PR and README file
- ✅ Correct Linters

However, there is always room for improvement.

## Required Changes 🛑

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
**Hi @ahmed025566 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge:shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights


✔️ Clean and well-organized code 
✔️ Professional readme file
✔️ Well done implementing the requested changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**